parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -285,7 +285,7 @@ extractHyperkube() {
 }
 
 extractKubeBinaries() {
-    KUBE_BINARY_URL=${KUBE_BINARY_URL:-"https://dl.k8s.io/v${KUBERNETES_VERSION}/kubernetes-node-linux-amd64.tar.gz"}
+    KUBE_BINARY_URL=${KUBE_BINARY_URL:-"https://kubernetesartifacts.azureedge.net/kubernetes/v${KUBERNETES_VERSION}/binaries/kubernetes-node-linux-amd64.tar.gz"}
```

Comment:
@andyzhangx @feiskyer is this mooncake-compatible?

Reply:
thanks.
`time wget https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.9/binaries/kubernetes-node-linux-amd64.tar.gz`
On global Azure, it's 17s, and on Mooncake it's 57s, it's still mooncake compatible, while on mooncake, download is a little slower(with 40s slower)
cc @karataliu @xinhe